src/ImageBackendFactory.php

```diff
@@ -35,7 +35,7 @@ public function __construct(Factory $creator)
      * @param array $params The constructor parameters.
      * @return object The created service instances.
      */
-    public function create($service, array $params = [])
+    public function create(string $service, array $params = []): ?Image_Backend
```

Comment:
Why would is this nullable?

Reply:
See lines 58 on - if the file is missing, null is returned.